openid/src/main/java/fish/payara/security/openid/AccessTokenIdentityStore.java

```diff
@@ -126,7 +126,7 @@ public BearerVerifier(OpenIdConfiguration configuration) {
         public void verify(JWTClaimsSet claims, SecurityContext c) throws BadJWTException {
             StandardVerifications standardVerifications = new StandardVerifications(configuration, claims);
 
-            standardVerifications.requireSameIssuer();
+            standardVerifications.requireIssuer(configuration.getProviderMetadata().getAccessTokenIssuerURI());
```

Comment:
Perhaps if configuration.getProviderMetadata().getAccessTokenIssuerURI() isEmpty
standardVerifications.requireSameIssuer()
else 
standardVerifications.requireIssuer(configuration.getProviderMetadata().getAccessTokenIssuerURI());

Reply:
That's actually solved in `OpenIdProviderMetadata`:

```
    public String getAccessTokenIssuerURI() {
        if (accessTokenIssuerURI == null) {
            return issuerURI;
        }
        return accessTokenIssuerURI;
    }
```